### PR TITLE
Improved error message when loading ServiceDiscovery impl

### LINKED ISSUE #28550: Misleading error message when load akka discovery implemenation
The following message gets logged when akka discovery is badly configured: 

```scala
java.lang.IllegalArgumentException: 
akka.discovery.akka.discovery.kubernetes-api.class must contain field `class` 
that is a FQN of a `akka.discovery.ServiceDiscovery` implementation
```

Instead it should say: 

> akka.discovery.kubernetes-api.class must contain a FQN of `akka.discovery.ServiceDiscovery` implementation

Note that `akka.discovery` appears twice and the message gives the impression that a nested `class` field is required.

### PR DESCRIPTION
**Pull request by octonato**
Fixes #28550

### CODE REVIEW

File: akka-discovery/src/main/scala/akka/discovery/Discovery.scala [hunk 1]
```diff
@@ -63,7 +63,7 @@ final class Discovery(implicit system: ExtendedActorSystem) extends Extension {
       if (config.hasPath(path)) config.getString(path)
       else
         throw new IllegalArgumentException(
-          s"$path must contain field `class` that is a FQN of a `akka.discovery.ServiceDiscovery` implementation")
+          s"$path must point to a FQN of a `akka.discovery.ServiceDiscovery` implementation")
```
Comment (renatocaval) [inline akka-discovery/src/main/scala/akka/discovery/Discovery.scala]: As explained in #28550, this message is misleading as it gives the impression that you need a nested `class` field. 

As we can see here: https://github.com/akka/akka/pull/28551/files#diff-131a7333ee160e8ff02426ecbac4ef22R81

The already append a `.class` to the configName.

**Review by octonato — COMMENTED** [review]
I'm convinced that the misleading message reported in #28550 was due to a bad configuration. 

However, we can still improve the message, hence this PR.

**Review by johanandren — APPROVED** [review]
Makes sense to me

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**